routes/v1.js

```diff
@@ -46,99 +93,131 @@ api.declare({
   deferAuth:      true,
   scopes:         ['aws-provisioner:create-worker-type:<workerType>'],
   input:          SCHEMA_PREFIX_CONST + 'create-worker-type-request.json#',
-  output:         SCHEMA_PREFIX_CONST + 'create-worker-type-response.json#',
+  output:         SCHEMA_PREFIX_CONST + 'get-worker-type-response.json#',
   title:          "Create new Worker Type",
   description: [
-    // Document what this method does
+    "Create a worker type and ensure that all EC2 regions have the required ",
+    "KeyPair"
   ].join('\n')
-}, function(req, res) {
+}, 
+function(req, res) {
   var ctx         = this;
   var input       = req.body;
   var workerType  = req.params.workerType;
```

Comment:
I know this sucks... and in future if we migrate to `hapi` this can be done at a higher level.
But I advice that we validate that `workerType` satisfies `/^[a-zA-Z0-9-_]{1,22}$/` at least when creating worker types...